README.org

```diff
@@ -377,17 +377,22 @@ feature that does not need synchronization, you can skip this step.
 
 **** WebDAV
 
-organice has support for WebDAV and ships with a Docker container with
-a WebDAV server based on Apache. You can make use of that and use this
-WebDAV back-end for local development.
+organice has support for WebDAV and you can make use of that for local development.
+We offer instructions for two options for setting up WebDAV servers but you
+may use any other standard solution. 
+
+- Option 1: A Docker Container with Apache
+  organice ships with a Docker container with a WebDAV server based on Apache.
+- Option 2: A WebDAV server in go (Docker image also available)
 
 Having said that, if you're a Dropbox or Google Drive user, then it's
```

Comment:
Oh, if you're just editing this file, you could ~strike~ the "or Google Drive" since support has been dropped.

Reply:
:O

Thanks for the feedback!